samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -94,7 +94,7 @@ tests:
     integration_platforms:
       - nrf5340dk/nrf5340/cpuapp/ns
     platform_allow: nrf5340dk/nrf5340/cpuapp/ns
-    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
+    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek_nrf7000
```

Comment:
This is a connectivity build, not a scan-only build, so I think that means we should stick with `nrf7002ek`, right?